dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -483,6 +483,27 @@ public Response setAggregatedMetrics(String id, String versionId, Partner platfo
         return Response.ok().entity(version.getMetricsByPlatform()).build();
     }
 
+    @Override
+    public Map<Partner, Metrics> getAggregatedMetrics(String id, String versionId) {
+        Entry<?, ?> entry;
+        try {
+            entry = getEntry(id, Optional.empty());
```

Comment:
Re: `Optional<User> user` comment, should pass it along here so it can verify that the user can read the workflow